docs/renarde/JavaSupport.md

```diff
@@ -0,0 +1,14 @@
+# Java Support
+
+### REST Endpoints
+
+For Renarde `Controller` classes, Renarde uses a different strategy than JAX-RS to determine which methods count as REST endpoints, and what the path of each endpoint is.
+vscode-quarkus contains functionality so that the existing JAX-RS features (code lens and workspace symbols) are handled according to these rules in `Controller` classes.
+
+![](./images/renarde-code-lens.png)
+
+*JAX-RS code lens in a Renarde application*
+
+![](./images/renarde-workspace-symbols.png)
```

Comment:
Add a little description which explains that codelens displays URL by using class name, etc and that:

 * codelens must be activated with settings
 * codelens appears only when the application is started